lib/l10n/app_th.arb

```diff
@@ -1,3 +1,5 @@
 {
-  "hello": "สวัสดี"
+    "emailInputHint": "",
+    "passwordInputHint": "",
+    "loginButton": ""
```

Comment:
Should we remove these `app_th.arb` and `app_vi.arb` files as we only use English version only?